mixer/pkg/runtime/config/ephemeral.go

```diff
@@ -204,6 +204,24 @@ func (e *Ephemeral) processHandlerConfigs(counters Counters, errs *multierror.Er
 	for key, resource := range e.entries {
 		var info *adapter.Info
 		var found bool
+
+		if key.Kind == constant.HandlerKind {
+			handlerProto := resource.Spec.(*config.Handler)
+			staticConfig := &HandlerStatic{
+				Name:    key.Name,
+				Adapter: e.adapters[handlerProto.Adapter],
+			}
+
+			if handlerProto.Params != nil {
```

Comment:
@guptasu here, the handlerProto.Params type is `map[string]interface {}`. staticConfig.Params takes a `proto.Message`. What is the recourse here?